src/Akka.Streams.Kafka/Stages/Consumers/Abstract/BaseSingleSourceLogic.cs

```diff
@@ -232,9 +235,8 @@ protected virtual void MessageHandling((IActorRef, object) args)
 
                     break;
 
-                case Terminated terminated:
-                    if (Log.IsInfoEnabled)
-                        Log.Info("Consumer actor terminated: {0}", terminated.ActorRef.Path);
```

Comment:
Second big fix: this is probably the source of several supervision bugs, but the death pact between the stream stage and its consumer actor needs to be honored

Reply:
This is probably where a bunch of our supervision errors came from